instant_news/lib/api/NewsApi.dart

```diff
@@ -12,4 +12,13 @@ class NewsApi {
     Response response = await BaseDio().dio.get("everything", queryParameters: param);
     return NewsResponse.fromJson(response.data);
   }
-}
\ No newline at end of file
+
+  Future<NewsResponse> getTrending(String country) async {
+    final Map<String, String> param = {
+      'country': country,
+      'apiKey': BaseDio().apiKey,
```

Comment:
can we move it to the repository? add the apiKey when send out the request?

Reply:
okie~ will resolve this in next mr